support-frontend/assets/pages/supporter-plus-landing/setup/setUpRedux.ts

```diff
@@ -169,6 +170,9 @@ export function setUpRedux(store: ContributionsStore): void {
 	);
 	selectInitialAmounts(state, dispatch, contributionType);
 
+	const billingPeriod = contributionType === 'ANNUAL' ? 'Annual' : 'Monthly';
+	dispatch(setBillingPeriod(billingPeriod));
```

Comment:
This needs to be initially setup otherwise when you refresh the page the state is lost to redux.


https://github.com/guardian/support-frontend/assets/31692/19d2ca9b-e4ab-4be1-a010-f4851ff7313a